src/PluginClient.php

```diff
@@ -50,8 +50,6 @@ final class PluginClient implements HttpClient, HttpAsyncClient
      *
      *     @var int      $max_restarts
      * }
-     *
-     * @throws \RuntimeException if client is not an instance of HttpClient or HttpAsyncClient
```

Comment:
This doc was incorrect anyway, so I think my change is fine, and can be regarded as a bug fix.

NB In general, I don't think we ever need throws docs for "Errors", only "Exceptions", so this is why I've not added any. One can think of Errors in PHP just like how Java's type system treats RuntimeExceptions.

Reply:
i agree :+1: